### PR TITLE
fix: Raise the tensorflow base container size limit

### PR DESCRIPTION
**Pull request by math411**
*Issue #, if available:*

*Description of changes:*
fix: Raise the tensorflow base container size limit

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/amazon-braket-containers/blob/main/CONTRIBUTING.md) doc
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/amazon-braket-containers/blob/main/CONTRIBUTING.md#commit-your-change)
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-braket-containers/blob/main/README.md) and [API docs](https://github.com/aws/amazon-braket-containers/blob/main/CONTRIBUTING.md#documentation-guidelines) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tensorflow/buildspec.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ images:
   BuildTensorflowGPUJobsPy3DockerImage:
     <<: *JOBS_REPOSITORY
     build: &TENSORFLOW_GPU_JOBS_PY3 false
-    image_size_baseline: 8014
+    image_size_baseline: 9400
```
Comment (krneta) [inline tensorflow/buildspec.yml:27]: Why did it go up?

Comment (math411) [reply tensorflow/buildspec.yml:27]: We were close to the limit before and the new update grew the container.Seems the base container grew in size.

Comment (krneta) [reply tensorflow/buildspec.yml:27]: I don't mean to harp on this, but 1.4 GB is a lot for the base container to grow. Do you know what it was?

Comment (math411) [reply tensorflow/buildspec.yml]: Limited the size to more closely follow the build

**Review by krneta — APPROVED** [review]

**Pull request merged**